dll/win32/advapi32/misc/evntrace.c

```diff
@@ -0,0 +1,42 @@
+/*
+ * PROJECT:     ReactOS system libraries
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     advapi32.dll Event tracing stubs
+ * COPYRIGHT:   Copyright 2017 Mark Jansen (mark.jansen@reactos.org)
+ */
+
+#include <wmistr.h>
+#include <evntrace.h>
+
+WINE_DEFAULT_DEBUG_CHANNEL(advapi);
+
+
+TRACEHANDLE
+WINAPI
+OpenTraceA(IN PEVENT_TRACE_LOGFILEA Logfile)
+{
+    UNIMPLEMENTED;
+    SetLastError(ERROR_ACCESS_DENIED);
+    return INVALID_PROCESSTRACE_HANDLE;
+}
+
+TRACEHANDLE
+WINAPI
+OpenTraceW(IN PEVENT_TRACE_LOGFILEW Logfile)
+{
+    UNIMPLEMENTED;
+    SetLastError(ERROR_ACCESS_DENIED);
+    return INVALID_PROCESSTRACE_HANDLE;
+}
+
+ULONG
+WINAPI
+ProcessTrace(IN PTRACEHANDLE HandleArray,
+             IN ULONG HandleCount,
+             IN LPFILETIME StartTime,
+             IN LPFILETIME EndTime)
+{
+    UNIMPLEMENTED;
+    return ERROR_NOACCESS;
```

Comment:
Is it the best value to return? (Wine returns ERROR_CALL_NOT_IMPLEMENTED here for example.)

Reply:
I looked the error codes of this function up, and this one was documented as `An exception occurred in one of the callback functions that receives the events.`